tests/Transport/TransportBenchmarkTest.php

```diff
@@ -18,8 +18,24 @@
  */
 class TransportBenchmarkTest extends BaseTest
 {
+    /**
+     * @var int
+     */
     protected $_max = 1000;
+
+    /**
+     * @var int
+     */
     protected $_maxData = 20;
+
+    /**
+     * @var array<string, array<string, array{
```

Comment:
Is this auto generated or manual? Same question for the other more complex blocks?

Reply:
This is manual, as phpstan is not that "smart" :)